webapps-schema/src/main/java/io/camunda/webapps/schema/descriptors/usermanagement/index/TenantIndex.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.webapps.schema.descriptors.usermanagement.index;
+
+import io.camunda.webapps.schema.descriptors.usermanagement.UserManagementIndexDescriptor;
+
+public class TenantIndex extends UserManagementIndexDescriptor {
+  public static final String INDEX_NAME = "tenants";
+  public static final String INDEX_VERSION = "8.7.0";
+
+  public static final String TENANT_KEY = "tenantKey";
+  public static final String TENANT_ID = "tenantId";
+  public static final String NAME = "name";
+  public static final String ENTITY_KEY = "entityKey";
+  public static final String ENTITY_TYPE = "entityType";
```

Comment:
🔧 Sorry @Oleksiivanov this is my fault, in the ES indexes we will want a field that contains a list of the assigned entities, for example [this is the same in roles](https://github.com/camunda/camunda/pull/23864/files#diff-b1529f330014dfe26200f492ceae14fc9c3999d18607958db41fa2d2e6356ca1R18), the record fields contain these two you have here as a way to provide members to be assigned to the tenant.

Reply:
Its possible that these can be deleted so I'd leave them as they are for now :)